src/helpers.ts

```diff
@@ -70,3 +70,16 @@ export function updatePropertiesRecursive(
   });
   return obj;
 }
+
+/**
+ * Calculate skip and limit options from given page and
+ * pageSize. If value is not given, use default values:
+ * 0 for page and 100 for pageSize
+ */
+export function calculateSkipAndLimit(p?: number, pageSize?: number) {
+  const requestedPage = Number(p) >= 0 ? Number(p) : 0;
```

Comment:
Is there particular reason why we cast typed variables to numbers before the check?